template.rb

```diff
@@ -45,14 +45,23 @@ def apply_template!(template_root)
   apply 'config/template.rb'
   apply '.gitignore.rb'
 
+  if ENV['CI']
+    @add_devise = true
+    apply '.template/addons/devise/template.rb'
+  elsif yes?("Would you like to add Devise gem?")
```

Comment:
I think we can use `install_addon_prompt` now after rebasing with `develop`

Reply:
Yeah, already done cc7b7c1 ✌️